components/collector/src/source_collectors/sonarqube/security_warnings.py

```diff
@@ -19,19 +19,17 @@ async def _landing_url(self, responses: SourceResponses) -> URL:
         base_landing_url = await SourceCollector._landing_url(self, responses)  # pylint: disable=protected-access
         component = self._parameter("component")
         branch = self._parameter("branch")
+        common_url_parameters, extra_url_parameters = f"?id={component}&branch={branch}", ""
         if "vulnerability" in security_types and "security_hotspot" in security_types:
-            landing_url = f"{base_landing_url}/dashboard?id={component}&branch={branch}"
+            landing_path = "dashboard"
         elif "vulnerability" in security_types:
-            landing_url = f"{base_landing_url}/project/issues?id={component}&resolved=false&branch={branch}"
+            landing_path = "project/issues"
+            extra_url_parameters = f"{self._query_parameter('severities')}&resolved=false&types=VULNERABILITY"
```

Comment:
should hardcoded `types=VULNERABILITY` not be combined with `_query_parameter(self.types_parameter)`?

Reply:
or is that automagically done by order of combining in `sonarqube/violations.py`.. hmm probably good to add a comment

Reply:
Using `_query_parameter(self.types_parameter)` would result in `&security_types=...` but we need `&types=...`. I'll add a comment.